packages/sdk-client/test/auth.test/base-auth-flow.test.ts

```diff
@@ -455,6 +455,7 @@ describe('Base Auth Flow', () => {
       // await promise failure which should set `requestState` to `false`
       await startCreateBaseMiddleware()
     } catch (error) {
+      expect(requestCount).toBe(1) // Make sure that nock runned the mocked request
```

Comment:
Why was this moved to the catch blocked?

Reply:
1. It actually makes more sense. This test is named `when createAuthMiddlewareBase fails`, so the verification should be in catch if possible.
2. Before we were using `log` to increase the requestCount, but this method is now removed. I have to move this requestCount increasing to `post`. However, this increasing now works at a different moment than with `log`, so this expect cannot be at its original position anymore.